### PR TITLE
DiscardArea always displayed when playing

### PR DESCRIPTION
**Pull request by sarahmamy**
What I did
* Discard is now always displayed on screen, right next to the Turns History
* Made Tokens smaller
* Put bold on Cards Text
* Made discard cards smaller
* Cleaning on `play.tsx`
* Remove commented code

### CODE REVIEW

File: src/components/card.tsx [hunk 1]
```diff
@@ -158,7 +158,7 @@ export default function Card(props: Props) {
     >
       {/* Card value */}
       <Txt
-        className={classnames(`txt-${color}-dark`, {
+        className={classnames(`b txt-${color}-dark`, {
```
Comment (sarahmamy) [inline src/components/card.tsx:161]: Displaying text in bold on cards

File: src/components/gameBoard.tsx [hunk 2]
```diff
@@ -16,50 +14,64 @@ import {
 import { useGame } from "~/hooks/game";
 
 interface Props {
-  onSelectDiscard: Function;
   onMenuClick: Function;
-  onShowRollback: Function;
+  onRollbackClick: Function;
```
Comment (sarahmamy) [inline src/components/gameBoard.tsx:18]: Just to have same naming for Menu and Rollback

File: src/components/gameBoard.tsx [hunk 3]
```diff
@@ -4,9 +4,7 @@ import { CardWrapper } from "~/components/card";
 import HomeButton from "~/components/homeButton";
```
Comment (sarahmamy) [inline src/components/gameBoard.tsx:4]: In game Board, I removed the DiscardPile. It is now close to the Instructions area

File: src/pages/play.tsx [hunk 4]
```diff
@@ -553,7 +556,7 @@ export default function Play() {
 
           {!interturn && (
             <div className="flex flex-column">
-              <div className="h-100 overflow-y-scroll">
+              <div className="h-100 ">
```
Comment (sarahmamy) [inline src/pages/play.tsx:559]:
```suggestion
              <div className="h-100">
```

File: src/components/turn.tsx [hunk 5]
```diff
@@ -69,6 +78,7 @@ export default function Turn(props: Props) {
           {" & drew "}
           <Card
             card={turn.card}
+            className="ml1 mr1" // When we'll add A, B, C
```
Comment (bstnfrmry) [inline src/components/turn.tsx:81]: This will indeed conflict with the position but not a big deal.

File: src/components/turn.tsx [hunk 6]
```diff
@@ -19,20 +19,27 @@ export default function Turn(props: Props) {
...
   return (
-    <div className="di mb2 pre">
+    <div className="dib">
       {includePlayer && (
         <>
-          <PlayerName player={game.players[turn.action.from]} />
-          <Txt value=" " />
+          <Txt className="inline-flex items-center gray" value="-" />
+          <PlayerName
+            className="mr1 ml1"
```
Comment (bstnfrmry) [inline src/components/turn.tsx:27]: You can use `mh1` (`h` for horizontal) instead of using left & right margins.

File: src/components/turn.tsx [hunk 7]
```diff
@@ -19,20 +19,27 @@ export default function Turn(props: Props) {
   const game = useGame();
 
   return (
-    <div className="di mb2 pre">
+    <div className="dib">
       {includePlayer && (
         <>
-          <PlayerName player={game.players[turn.action.from]} />
-          <Txt value=" " />
+          <Txt className="inline-flex items-center gray" value="-" />
```
Comment (bstnfrmry) [inline src/components/turn.tsx:25]: Looks nice! 👌

File: src/pages/play.tsx [hunk 8]
```diff
@@ -473,66 +452,90 @@ export default function Play() {
...
+                <Lobby
+                  onAddBot={onAddBot}
+                  onJoinGame={onJoinGame}
+                  onStartGame={onStartGame}
+                />
+              ) : (
+                <div className="h4 pt0-l overflow-y-scroll">
+                  {game.status === IGameStatus.OVER &&
+                    (isReplayMode(game) ? (
+                      <ReplayViewver
```
Comment (bstnfrmry) [inline src/pages/play.tsx:483]: `ReplayViewer`* (typo ;)

File: src/components/hint.tsx [hunk 9]
```diff
@@ -17,7 +18,12 @@ export default function Hint(props: Props) {
 
   return (
     <>
-      <div className="flex justify-center items-center hint">
+      <div
+        className={classnames(
+          props.className,
```
Comment (bstnfrmry) [inline src/components/hint.tsx:23]: For style consistency, can you extract `{ className }` from `props` at the very beginning of the component? Like we do in the other ones.

File: src/pages/play.tsx [hunk 10]
```diff
@@ -417,13 +403,6 @@ export default function Play() {
     });
   }
 
-  function onSelectDiscard() {
-    onSelectArea({
-      id: "discard",
-      type: ActionAreaType.DISCARD
```
Comment (bstnfrmry) [inline src/pages/play.tsx:423]: Let's also remove the `ActionAreaType.DISCARD` option from the enum list (defined here https://github.com/bstnfrmry/hanabi/blob/master/src/components/actionArea.ts#L22)

**Pull request merged**